internal/dnfjson/dnfjson.go

```diff
@@ -422,6 +454,21 @@ type Request struct {
 	Arguments arguments `json:"arguments"`
 }
 
+// Hash returns a hash of the unique aspects of the Request
+func (r *Request) Hash() string {
+	h := sha256.New()
+	h.Write([]byte(r.Command))
```

Comment:
```suggestion
	_ = h.Write([]byte(r.Command))
```

feels slightly ugly but it's probably the easiest fix.

Reply:
Yeah, Write actually cannot return an error but it also returns the length of the input (which it also doesn't change) so it's cleaner to just use `//nolint:errcheck` on the whole block.